src/e2e/pages/authPage.ts

```diff
@@ -71,4 +71,34 @@ export class AuthPage {
     expect(verificationCode).toBeDefined();
     await this.enterVerificationCode(verificationCode as string);
   }
+
+  async signInToFxA(email: string, password: string) {
+    await this.page.goto(process.env.FXA_SETTINGS_URL as string);
+    await this.page.context().clearCookies();
+    await this.page
+      .locator("//input[@type='password'] | //div/input[@type='email']")
+      .waitFor({ state: "visible" });
+    const visible = await this.useDifferentEmailButton.isVisible();
+    if (visible) {
+      await this.useDifferentEmailButton.click();
+      await this.page.waitForURL(/^(?!.*signin).*/);
+    }
+
+    // enter email
+    await this.emailInputField.fill(email);
+    await this.continueButton.click();
+    await this.page.waitForURL(/^(?!.*signin).*/);
+
+    // enter password
+    await this.passwordInputField.fill(password);
+    await this.continue({ waitForURL: process.env.FXA_SETTINGS_URL });
+  }
+
+  async initSilentAuth() {
+    await this.page.goto(
+      `${process.env.E2E_TEST_BASE_URL as string}/?feature_flags=PromptNoneAuthFlow&utm_source=moz-account&utm_campaign=settings-promo&utm_content=monitor-free`,
```

Comment:
So as per my other comment, this [would be](https://playwright.dev/docs/api/class-page#page-set-extra-http-headers):

```suggestion
    await this.page.setExtraHTTPHeaders({ "x-forced-feature-flags": "PromptNoneAuthFlow" });
    await this.page.goto(
      `${process.env.E2E_TEST_BASE_URL as string}/?utm_source=moz-account&utm_campaign=settings-promo&utm_content=monitor-free`,
```

Reply:
Thanks! Update in [7f3feea](https://github.com/mozilla/blurts-server/pull/5150/commits/7f3feeaca6ffce562b3ab32e7319ebe97b26fda4).